Software Packages (#369)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/packages/resources/golang/_example_package_webhook_config.mdx

```json showLineNumbers
{
  "identifier": "goDependencyMapper",
  "title": "Go Dependency Mapper",

>   "description": "A webhook configuration to map Go dependencies to Port",

Reviewer: ```suggestion
  "description": "A webhook configuration to map packages and dependencies from a file,
```
